triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -276,6 +276,13 @@ object Server {
           }
         )
       },
+      // List all running triggers
+      get {
+        path("list") {
+          val allTriggers = triggers.keys.mkString(",")
+          complete(s"Triggers currently running: $allTriggers")
```

Comment:
This should really return JSON not some random string format (which holds for all endpoints but the other ones mostly don’t return meaningful information anyway or just have a single string like `start`)

Reply:
Agreed

Reply:
Now returns a JSON list with the trigger ids. Assuming it doesn't need more structure than that